LectureNotes/Lectures/Fall2018/Lecture6/ASPlecturenotes.tex

```diff
@@ -0,0 +1,142 @@
+    \end{aligned}$$
+\end{enumerate}
+
+\noindent Thus, under the conditions of step 2, when $N\sim PPP(m)$, we have that
+$$\begin{aligned}
+\mathbb{E}[P] &= \int_{T\times T \setminus diag(T)} \mathbb{E}[N(dx)N(dy)] \rightarrow \int_T \int_T P(|x-y|)m^2 dxdy\\
+ &= \int_T m^2 dx = m^2
+\end{aligned}$$
+
+\noindent So: $\frac{P}{m^2}\approx 1 $ for a uniform PPP.
+
+%\subsection*{Recall:} We have points ${x_1,...,x_m}$ estimate of local density around $x_i$ on T. Are they Poisson?
+%\noindent\textbf{Strategy:} 
+%
+%$$\begin{aligned}
+%\text{Let } P_i = & \sum_{j\neq i} \rho (|x_i -x_j|) \text{ and }  P = \sum_i P_i \text{ for } e(r) \text{ in } \int_T\rho(|r|)dr=1\\
+%\text{ If } N\sim  PPP&  \text{ on } T \text{ with uniform intensity } m \text{ then } 
+%\mathbb{E}[P] = \int_{T\times T \setminus diag(T)} \mathbb{E}[N(dx)N(dy)] \\
+%\text{ and } \mathbb{E}&[N(dx)N(dy)] = m^2dxdy \text{ for } x\neq y\\
+%\end{aligned}$$
+
+\subsection*{Aside:}
+In general, if it is not a Poisson process in the above equations we would have that: 
+$$\begin{aligned}
+\mathbb{E}[N(dx)N(dy)] = m^2dxdy +  cov[N(dx),N(dy)] = m^2g(x,y)dxdy
+\end{aligned}$$
+
+where $g$ is the correlation function such that:
+$g(x,y) = \lim_{\varepsilon \to 0} \dfrac{1}{\pi^2\varepsilon^4}cor[\beta_\varepsilon(x),\beta_\varepsilon(x_2)]$. Then,
+
+$$\begin{aligned}
+\int_{T\times T \setminus diag(T)}\mathbb{E}[N(dx)N(dy)] &\rightarrow \int_Tm^2dx + \int_T\int_T\rho(|x-y|)g(x,y)m^2dxdy \\
+&=  m^2 + \text{SOMETHING ELSE}
+\end{aligned}$$
+
+The ``SOMETHING ELSE" will be bigger if points tend to be clustered on the scale determined by $p$, and smaller if points repels each other on that scale.
+
+\subsection*{How far off do we expect?}
+
+$$\begin{aligned}
+var[P] &= cov[P,P] \\
+&= cov[\int_{T\times T \setminus diag(T)}\rho(|x-y|) N(dx)N(dy), \int_{T\times T \setminus diag(T)}\rho(|u-v|)N(du)N(dv)] \\ 
+&= \int_A\rho(|x-y|)\rho(|u-v|) cov[N(dx)N(dy),N(du)N(dv)]
+\end{aligned}$$
+
+Let $A=(x,y,u,v) \in T^4 : x\neq y, u\neq v$. Then we have the following cases:
+
+$$\begin{cases}
+x\notin \{u,v\}, y\notin \{u,v\} &\Rightarrow cov[N(dx) N(dy), N(du)N(dv)] = 0\\
+x=u, y\notin\{u,v\} 		  &\Rightarrow cov[-,-] = \mathbb{E}[N(dy)]\mathbb{E}[N(du)]var[N(dx)] = m^3dxdydv\delta_x(du)\\
+x=u,y=v, (x\neq y)		  &\Rightarrow cov[-,-] = m^2dxdy \delta_x(du)\delta_y(dv)\\
+\end{cases}$$
+
+Which implies that:
+\
+$$\begin{aligned}
+var[P] &= 4\int_T \int_T\int_T \rho (|x-y|) \rho(|x-v|)m^3 dxdydv + 2\int_T\int_T\rho (|x-y|)^2 m^2dxdy\\
+	&= \mathbf{A} + \mathbf{B}
+\end{aligned}$$
+
+But what does that mean?
+
+Let $\rho(r) = \begin{cases} \frac{1}{\pi R^2} \text{, if } r<R  \\  0, \text{  else.} \end{cases}$. Then $\mathbf{A} = 4m^3$ and $\mathbf{B}=\frac{2m^2}{\pi R^2}$.
+
+So, $var[\frac{P}{M^2}] \approx \frac{4}{m} + \frac{2}{\pi R^2 m^2} $.
+
+Thus, to determine if the bombs are being dropped at random (i.e. via a PPP), we can use the statistic:
```

Comment:
in the case that the bombs are falling on the torus of unit area and $\rho$ is uniform on a circle